pkg/rc/farm.go

```diff
@@ -13,10 +13,12 @@ import (
 	"github.com/square/p2/pkg/kp/rcstore"
 	"github.com/square/p2/pkg/labels"
 	"github.com/square/p2/pkg/logging"
+	p2metrics "github.com/square/p2/pkg/metrics"
```

Comment:
We could name our package p2metrics, but I don't feel strongly enough to insist on the change here